<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_LaserBeam">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LaserBeamAlignment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LaserPlane">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LaserPlaneAlignment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LaserVerification">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LaserVerification</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Laser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>180</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Laser Select</string>
    </property>
    <widget class="QRadioButton" name="radioButton_L1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_L2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_L3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_L4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Distance">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>150</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Distance Select</string>
    </property>
    <widget class="QRadioButton" name="radioButton_D1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_D2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SerialNum">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Laser Serial Number</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_L4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SerialNum_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>131</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fixture Number</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_F4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_F3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_F2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_F1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>pushButton_LaserBeam</zorder>
   <zorder>pushButton_LaserPlane</zorder>
   <zorder>groupBox_Laser</zorder>
   <zorder>groupBox_Distance</zorder>
   <zorder>groupBox_SerialNum</zorder>
   <zorder>groupBox_SerialNum_2</zorder>
   <zorder>pushButton_LaserVerification</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLaser_Alignment_Tool">
    <property name="title">
     <string>Laser Assembly Tool</string>
    </property>
   </widget>
   <addaction name="menuLaser_Alignment_Tool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
